flyteplugins/go/tasks/pluginmachinery/flytek8s/container_helper_test.go

```diff
@@ -237,19 +237,37 @@ func TestApplyResourceOverrides_OverrideGpu(t *testing.T) {
 	gpuRequest := resource.MustParse("1")
 	overrides := ApplyResourceOverrides(v1.ResourceRequirements{
 		Requests: v1.ResourceList{
-			resourceGPU: gpuRequest,
+			ResourceNvidiaGPU: gpuRequest,
```

Comment:
Is it safe to make this change? IIUC the arguments to `ApplyResourceOverrides` will have "gpu" rather than "nvidia/gpu" currently?